core/core.go

```diff
@@ -291,6 +319,7 @@ func NewTextile(ctx context.Context, conf Config) (*Textile, error) {
 			upb.RegisterAPIServer(t.server, us)
 		}
 		bpb.RegisterAPIServer(t.server, bs)
+		ffsRpc.RegisterRPCServiceServer(t.server, &ffsService)
```

Comment:
There is some logic in gRPC that runs before the interceptors that determines if the request is supported by the server. You have to register _some_ FFS service if you want to be able to deal with FFS requests in the interceptor layer, so I just register the generated `UnimplementedRPCServiceServer` and we don't have to create our own stub in any way